pkgs/applications/graphics/pencil/default.nix

```diff
@@ -89,7 +89,7 @@ stdenv.mkDerivation rec {
     chmod a+x $out/opt/Pencil/libffmpeg.so
 
     # fix missing libudev
-    ln -s ${udev}/lib/systemd/libsystemd-shared.so $out/opt/Pencil/libudev.so.1
+    ln -s ${eudev}/lib/libudev.so.1 $out/opt/Pencil/libudev.so.1
```

Comment:
Would `ln -s ${systemd.lib}/lib/libudev.so.1  $out/opt/Pencil/libudev.so.1` also work?

Reply:
Maybe, is it better ?

Reply:
`eudev` is the gentoo-maintained `udev` fork while `systemd` is the canonical `udev` upstream.

Reply:
@adisbladis Ah this explains why I thought it is the proper one :D (I used gentoo for a very long time in my life)

@Mic92 Yes it works, in a few minutes I'll push the changes.